test/releases/99.0.0.go

```diff
@@ -24,7 +24,7 @@ func init() {
 				ParentName: "kube-cni-linux-bridge-plugin",
 				ParentKind: "DaemonSet",
 				Name:       "cni-plugins",
-				Image:      "quay.io/kubevirt/cni-default-plugins@sha256:d211830c45bde8608befb04cf2accc91490467f3b8e091b812b76453572c9ab8",
+				Image:      "quay.io/kubevirt/cni-default-plugins:v0.9.1-93-ge010479",
```

Comment:
the output of `make bump-linux-bridge` should produce a `@sha256` digest..

Reply:
It produces this one. You can also see this in the prev error on this CI run: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/1016/pull-cluster-network-addons-operator-unit-test/1437398234313003008